.eslintrc.js

```diff
@@ -3,6 +3,7 @@
 module.exports = {
   env: {
     es6: true,
+    node: true,
```

Comment:
TODO: clean up this file to remove the `extension/**` directives.

Reply:
Cleaned! 🧹